cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -1129,6 +1153,15 @@ profiles:
 			expectedError: `unknown field "foo"`,
 			checkErrFn:    runtime.IsStrictDecodingError,
 		},
+		{
```

Comment:
Actually, we're going to mark v1beta2 as deprecated after https://github.com/kubernetes/kubernetes/pull/110534, so I was doubt about whether we need this any more. -_-

Reply:
basically, this pr add some testcase for v1beta3, since it was on v1beta2 only, I think this should be okay.